src/functions/Deck.js

```diff
@@ -0,0 +1,26 @@
+const VALUES = ['1','2','3','4','5','6','7','8','9','10','J','Q','K']
+const SUITS = ['h','c','s','d']
+
+export const buildDeck = () => {
```

Comment:
Ao adicionar parâmetros a gente consegue fazer o `buildDeck` montar outros tipos de baralhos e com outras quantidades, não é tão útil para o que vocês precisam fazer depois, porém vai facilitar os testes e fica mais extensível, já que nem sempre precisaria buildar um deck com 52 cartas. O que acha?
```suggestion
export const buildDeck = (suits = SUITS, values = VALUES) => {
```

```js
const deck = buildDeck(['a', 'b', 'c'], ['1', '2', '3'])

expect(deck).toEqual([
  { suit: 'a', value: 1 },
  { suit: 'a', value: 2 },
  { suit: 'a', value: 3 },
  { suit: 'b', value: 1 },
  { suit: 'b', value: 2 },
  { suit: 'b', value: 3 },
  { suit: 'c', value: 1 },
  { suit: 'c', value: 2 },
  { suit: 'c', value: 3 },
])
```